src/definition/users/IUser.ts

```diff
@@ -16,4 +16,9 @@ export interface IUser {
     createdAt: Date;
     updatedAt: Date;
     lastLoginAt: Date;
+    active?: boolean;
+    joinDefaultChannels?: boolean;
+    verified?: boolean;
+    requirePasswordChange?: boolean;
+    sendWelcomeEmail?: boolean;
```

Comment:
I question whether this is something that needs to be a on a regular User. Will me updating a user after they've been created with these values result in them being set? For example, if I set `sendWelcomeEmail` on a user that has already been created will it send them an email? If not, then these properties which are only valid on the creation of a user should be on a specific interface for a creation user. So, something like `IUserCreation extends IUser` should work just find and only contain the properties. My two cents 🤷‍♂

Reply:
Indeed, the `sendWelcomeEmail` and `joinDefaultChannels` fields definitely do not belong on the `User` schema; they're actually parameters used by the user creation method, which makes them more suited to live in the [`IUserCreationOptions`](https://github.com/RocketChat/Rocket.Chat.Apps-engine/pull/195/files#diff-8f3d6315de545e85084bdfe362dff717R3) interface

Reply:
Thanks! Already changed them as you suggested :)